student-info-system/schooladmin/tables.py

```diff
@@ -113,3 +113,20 @@ class Meta:
         fields = ('major', 'catalogNumber', 'title', 'credits_earned')
         attrs = {"class": 'bcourse'}
         row_attrs = {'class': 'crow', 'data-id': lambda record: record.pk}
+
+
+class SemestersTable(tables.Table):
+    semester = tables.Column(attrs={'th': {'style': 'text-align: center;'}})
```

Comment:
maybe create a variable that holds the {'style': 'text-align: center;'} object since it is used for each one of these.

Reply:
Should I do that in a new task?

Reply:
I'd actually say don't worry about this because it might actually make it harder to read/understand now that I've looked at some of bruce's code.